mutiny-core/src/sql/glue.rs

```diff
@@ -0,0 +1,566 @@
+#![allow(dead_code, unused_variables, unused_imports, unreachable_code)] // TODO remove
```

Comment:
@elsirion could you check the approach for this `glue.rs` file whenever you have a chance?

The approach I've taken for the fedimint traits is to have a KV table on top of gluesql (w/ indexeddb under the hood) where each fedimint client will have their own row. I'm just doing a dump of the values on `commit` like you are with webimint. I tried a few different approaches but came back to that one being the safest while still adhering to the trait requirements.